pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/FileUploadService.java

```diff
@@ -285,6 +291,32 @@ public int removeUnusedUploads(Handle handle, Instant olderThanTimestamp, Intege
         return uploadIdsToDelete.size();
     }
 
+    public void sendNotifications(final Handle handle) {
+        StreamEx.of(handle.attach(FileUploadDao.class).findWithoutSentNotification())
+                .groupingBy(FileUpload::getStudyId)
+                .forEach((studyId, fileUploads) -> sendNotifications(handle, studyId, fileUploads));
+    }
+
+    private void sendNotifications(final Handle handle, final Long studyId, final List<FileUpload> fileUploads) {
+        final var study = handle.attach(JdbiUmbrellaStudy.class).findById(studyId);
+        if (StringUtils.isBlank(study.getNotificationEmail())) {
+            log.info("Study {} doesn't have an e-mail for notifications", study.getGuid());
+            return;
+        }
+
+        StreamEx.of(fileUploads)
+                .groupingBy(FileUpload::getParticipantUserId)
+                .forEach((participantId, uploads) -> sendNotification(study, participantId, uploads));
+
+        handle.attach(FileUploadDao.class).setNotificationSentByStudyId(studyId);
+        log.info("Notifications sent for {} study", study.getGuid());
+    }
+
+    private void sendNotification(final StudyDto study, final Long participantId, final List<FileUpload> fileUploads) {
+        //TODO: Send an e-mail somehow
+        log.info("EMAIL: A user #{} uploaded {} files in terms of {} study", participantId, fileUploads.size(), study.getGuid());
```

Comment:
For know I haven't found the existing solution and example of sending an e-mail. Should be simple, so please advice